specification/adp/resource-manager/Microsoft.AutonomousDevelopmentPlatform/preview/2021-02-01-preview/adp.json

```diff
@@ -786,37 +891,70 @@
       ]
     },
     "DataPoolLocation": {
+      "type": "object",
       "description": "Location of a Data Pool",
       "properties": {
         "name": {
           "description": "The location name",
           "type": "string"
+        },
+        "encryption": {
+          "description": "Encryption properties of a Data Pool location",
+          "$ref": "#/definitions/DataPoolEncryption"
         }
       },
       "required": [
         "name"
       ]
     },
+    "DataPoolEncryption": {
+      "type": "object",
+      "description": "Encryption properties of a Data Pool",
+      "properties": {
+        "keyVaultUri": {
```

Comment:
Can you add an example of using these properties?

Reply:
Added